resources/sql/ampache.sql

```diff
@@ -264,6 +264,53 @@ CREATE TABLE IF NOT EXISTS `catalog` (
 
 -- --------------------------------------------------------
 
+--
```

Comment:
drop the sql file from this as it's not relevant til i make a release and i do it from a zip install

Reply:
Just to be clear, I needed this file to test a clean install, but not submitted in the pull request?

Reply:
yeah, the sql file only gets updated for new people and it's sort of a new install only thing i need to worry about.

the update.php file is more important because i generate the sql file from a new install to create a default

Reply:
Ok,  I will make that change and some others when I have time today.